rare/components/tabs/games/game_info/game_info.py

```diff
@@ -215,184 +174,176 @@ def verify_result(self, app_name, success, failed, missing):
                 QMessageBox.Yes,
             )
             if ans == QMessageBox.Yes:
-                self.repair_game(igame)
+                self.repair_game(rgame)
 
     @pyqtSlot(str)
-    def move_game(self, dest_path):
-        dest_path = Path(dest_path)
-        install_path = Path(self.igame.install_path)
-        self.dest_path_with_suffix = dest_path.joinpath(install_path.stem)
-
-        if self.dest_path_with_suffix.is_dir():
-            self.existing_game_dir = is_game_dir(install_path, self.dest_path_with_suffix)
-
-        if not self.existing_game_dir:
-            for i in dest_path.iterdir():
-                if install_path.stem in i.stem:
-                    warn_msg = QMessageBox()
-                    warn_msg.setText(self.tr("Destination file/directory exists."))
-                    warn_msg.setInformativeText(
-                        self.tr("Do you really want to overwrite it? This will delete {}").format(
-                            self.dest_path_with_suffix
-                        )
+    def __on_move(self, dst_path: str):
+        """ This method is to be called from the button only """
+        new_install_path = os.path.join(dst_path, os.path.basename(self.rgame.install_path))
+
+        dir_exists = False
+        if os.path.isdir(new_install_path):
+            dir_exists = is_game_dir(self.rgame.install_path, new_install_path)
+
+        if not dir_exists:
+            for item in os.listdir(dst_path):
+                if os.path.basename(self.rgame.install_path) in os.path.basename(item):
```

Comment:
From what I gather, this seems to compare every entry in the `dst_path` folder, only to check if the basename of `self.rgame.install_path` is already present there. Could this be simplified to `os.path.exists(new_install_path)`?

Reply:
This is @invertedEcho 's code moved to `os.path` to `PathLib`. This is probably the result of conflicting requirements from the initial PR and I don't recall why it was done this way, pending a more in-depth look at some point.